### PR TITLE
Add 1/4 of a node option in simple menu

### LINKED ISSUE #20: Add a 1/4 of a node in Simple tab

### PR DESCRIPTION
**Pull request by t20100**
closes #20

### CODE REVIEW

**Comment by kif** [comment]
Well, I don't speak javascript ... but beside this, it looks ok

File: jupyterhub_moss/templates/option_form.html [hunk 1]
```diff
@@ -53,13 +53,24 @@
...
+      <label for="quarterCore">
+        <p>&frac14&nbsp;node</p>
+        <p id="quarterCpufield" class="label-extra-info"></p>
+      </label>
+
+      <input
+        type="radio"
+        id="halfCore"
+        name="nprocs_simple"
+        value="1"
```
Comment (loichuder) [inline jupyterhub_moss/templates/option_form.html:69]: Is `value` necessary here ? It will be overwritten by the JS part anyway, no ?

**Review by loichuder — APPROVED** [review]

File: jupyterhub_moss/templates/option_form.html (hunk 1 above)
Comment (t20100) [reply jupyterhub_moss/templates/option_form.html:69]: I don't recall why all fields have `value="1"`, it's most probably not needed.

Comment (t20100) [reply jupyterhub_moss/templates/option_form.html]: I removed them.

**Pull request merged**